Header header
float32[] Qdot_WF
float32[] Qdot_SF
float32[] Qdot_tang

